generator/config/stage/search.yaml

```diff
@@ -45,7 +45,7 @@ arguments:
         name: count
         optional: true
         type:
-            - string
+            - object
```

Comment:
Fixed bug.

Reply:
Should there be a JIRA ticket for this fix?

Reply:
This is not released yet.

Reply:
Thanks for the reminder. :smile:

In the future, including the original JIRA issue for the feature would be preferable to establish context from both the git history and JIRA issue (assuming the git bot is working again). Typically, the commits without JIRA links are usually CI fixes or Markdown edits.